pkg/disk/partition_table.go

```diff
@@ -1410,22 +1449,6 @@ func addPlainPartition(pt *PartitionTable, partition blueprint.PartitionCustomiz
 		}
 	}
 
-	var payload PayloadEntity
-	switch fstype {
-	case "swap":
```

Comment:
We should, IMHO, simply add an explicit case for `none` fstype and handle it accordingly.